packages/gatsby-theme-vtex/src/pages/index.tsx

```diff
@@ -6,7 +6,7 @@ import SuspenseViewport from '../components/Suspense/Viewport'
 import AboveTheFold from '../components/HomePage/AboveTheFold'
 import ErrorBoundary from '../components/Error/ErrorBoundary'
 import ErrorHandler from '../components/Error/ErrorHandler'
-import SEO from '../components/HomePage/SEO'
+import Seo from '../components/HomePage/SEO'
```

Comment:
Did you change this because of our linter? It can be called `SEO` now (all caps), just do a `yarn upgrade` on the root!

Reply:
Yes ! I will run `yarn upgrade` and rollback SEO casing then